docs/catalog/web/enable-text-compression.md

```diff
@@ -0,0 +1,46 @@
+---
+version: 1.0
+submitted_by: markus-ntt-seidl
+published_date: TBD
+category: web
+tags: 
+ - text
```

Comment:
_Text_ and _compression_ are not pre-defined tags. Please see here for the complete list- https://github.com/Green-Software-Foundation/green-software-patterns/blob/dev/docs/tags/web.md. How about _web_?

Reply:
Note: I have added the full list of suggested tags to the patterns website under the guide section: https://patterns.greensoftware.foundation/guide/suggested-tags